R/RMySQL.R

```diff
@@ -0,0 +1,10 @@
+#' @title RMySQL: Database Interface and MySQL Driver for R
+#' @name RMySQL
+#' @description Implements a DBI interface to MySQL and MariaDB databases.
+#'
+#' @seealso the \href{https://cran.r-project.org/web/packages/RMySQL/RMySQL.pdf}{package documentation}.
+#' @useDynLib RMySQL
+#' @importFrom Rcpp sourceCpp
+#' @docType package
+#' @aliases RMySQL RMySQL-package
+NULL
```

Comment:
No, literally `"_PACKAGE"`. Currently, title, description, and doctype are autogenerated (drawn from DESCRIPTION), everything else is still supported.